### PR TITLE
implemented anime search from openAPI

### PR DESCRIPTION
**Pull request by shaileshdeuex**

### CODE REVIEW

File: src/component/AnimeList.jsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
+const AnimeList = ({ apiData }) => {
+  if (apiData.status === 404) {
+    return <h1>{apiData.message}</h1>;
+  } else {
```
Comment (chirag-madlani) [inline src/component/AnimeList.jsx]: Should not handle API error here in the component. It should be handled where it get called

File: src/component/AnimeList.jsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    const { results } = apiData;
```
Comment (chirag-madlani) [inline src/component/AnimeList.jsx]: Instead of passing `apiData` props can we pass list directly?

File: src/index.css [hunk 3]
```diff
@@ -1,10 +1,17 @@
+*{
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
```
Comment (chirag-madlani) [inline src/index.css]: avoid this global CSS selector `*`

File: src/component/AnimeList.jsx (hunk 1 above)
Comment (shaileshdeuex) [reply src/component/AnimeList.jsx]: Implemented.

File: src/component/AnimeList.jsx (hunk 2 above)
Comment (shaileshdeuex) [reply src/component/AnimeList.jsx]: Implemented.

File: src/index.css (hunk 3 above)
Comment (shaileshdeuex) [reply src/index.css]: Implemented.

**Review by chirag-madlani — CHANGES_REQUESTED** [review]
1. Images are not aligned center vertically check image below
2. Give option to clear search
3. show loading when you are fetching data from the API
4. Add no result found message when API returns empty results

Screenshot 2021-03-22 at 4 04 06 PM

**Comment by shaileshdeuex** [comment]
> 1. Images are not aligned center vertically check image below
> 2. Give option to clear search
> 3. show loading when you are fetching data from the API
> 4. Add no result found message when API returns empty results
> 
> Screenshot 2021-03-22 at 4 04 06 PM

Implemented

File: src/App.js [hunk 4]
```diff
@@ -10,13 +10,25 @@ const apiRequest = async (search) => {
...
 
   const handleSubmite = async (event) => {
     event.preventDefault();
+    setgotData(false);
+    setdataLoading(true);
     const data = await apiRequest(animeSearch).then((res) => res);
     setApiData(data);
-    setgotData(true);
+    setdataLoading(false);
+    setanimeSearch("");
```
Comment (chirag-madlani) [inline src/App.js]: Text is getting clear automatically after results loaded

**Pull request merged**